packages/lib/src/api.ts

```diff
@@ -0,0 +1,15 @@
+import { init } from "@sesamy/sesamy-js";
+import type { Config } from "@sesamy/sesamy-js";
+
+export type Api = Awaited<ReturnType<typeof init>>;
```

Comment:
This should be exported from Sesamy-js..

Reply:
We had a global API in the type defs when sesamy-js was created. But I remember upgrading to a newer version once and it disappeared. Just forgot to poke you about it 😅

Yes, this would be great.